compiler/src/iree/compiler/Codegen/Common/test/verify_tuning_specs.mlir

```diff
@@ -47,22 +47,292 @@ module @foo_module attributes { transform.with_named_sequence } {
+
+// expected-error @+1{{Expected one named sequence with 'iree_codegen.tuning_spec_entrypoint', but found 2 (required by 'iree_codegen.tuning_spec_with_default_entrypoint')}}
+module @iree_default_tuning_spec attributes { iree_codegen.tuning_spec_with_default_entrypoint } {
+  transform.named_sequence @__kernel_config(%arg0: !transform.any_op {transform.consumed})
+      -> (!transform.any_op) attributes { iree_codegen.tuning_spec_entrypoint } {
+    transform.yield %arg0 : !transform.any_op
+  }
+
+  transform.named_sequence @main(%arg0: !transform.any_op {transform.readonly})
+      -> !transform.any_op attributes { iree_codegen.tuning_spec_entrypoint } {
+    transform.yield %arg0 : !transform.any_op
+  }
+}
+
+// -----
+
+module @iree_default_tuning_spec attributes { iree_codegen.tuning_spec_with_default_entrypoint } {
+    transform.named_sequence @match(%arg: !transform.any_op {transform.readonly}) -> (!transform.any_op) {
+      transform.yield %arg : !transform.any_op
+    }
+
+    transform.named_sequence @apply_op_config(%op: !transform.any_op {transform.readonly}) {
+        transform.yield
+    }
+
+    // expected-error @+1{{Unexpected op 'transform.print' in '__kernel_config' (required by 'iree_codegen.tuning_spec_with_default_entrypoint')}}
+    transform.named_sequence @__kernel_config(%arg0: !transform.any_op {transform.consumed})
+        -> (!transform.any_op) attributes { iree_codegen.tuning_spec_entrypoint } {
+        %res = transform.foreach_match in %arg0
+          @match -> @apply_op_config
+          : (!transform.any_op) -> (!transform.any_op)
+
+        transform.yield %res : !transform.any_op
+        transform.print {name = "Hello"}
+    }
+}
+
+// -----
+
+// expected-error @+1{{Expected one named sequence with 'iree_codegen.tuning_spec_entrypoint', but found 2 (required by 'iree_codegen.tuning_spec_with_default_entrypoint')}}
+module @iree_default_tuning_spec attributes { iree_codegen.tuning_spec_with_default_entrypoint } {
+  transform.named_sequence @__kernel_config(%arg0: !transform.any_op {transform.consumed})
+      -> (!transform.any_op) attributes { iree_codegen.tuning_spec_entrypoint } {
+    transform.yield %arg0 : !transform.any_op
+  }
+
+  module @extra_module attributes { iree_codegen.tuning_spec_entrypoint } {
+    transform.yield
+  }
+}
+
+// -----
+
+module @iree_default_tuning_spec attributes { iree_codegen.tuning_spec_with_default_entrypoint } {
+  transform.named_sequence private @dummy_func(!transform.any_op {transform.consumed}) -> !transform.any_op
+    transform.named_sequence @match(%arg: !transform.any_op {transform.readonly}) -> (!transform.any_op) {
+      transform.yield %arg : !transform.any_op
+    }
+
+    transform.named_sequence @apply_op_config(%op: !transform.any_op {transform.readonly}) {
+        transform.yield
+    }
+
+    // expected-error @+1{{Unexpected op 'transform.include' in '__kernel_config' (required by 'iree_codegen.tuning_spec_with_default_entrypoint')}}
```

Comment:
This is already covered by the previous test case that checks for print. We can drop this one.